insights/parsers/systemd_analyze.py

```diff
@@ -54,11 +54,33 @@ def parse_content(self, content):
         if not content:
             raise SkipException
 
+        unit_types = ['.service', '.socket', '.device', '.mount', '.automount', '.swap',
+                      '.target', '.path', '.timer', '.snapshot', '.slice', '.scope']
+
         for c in content:
-            time, service = c.split()
-            if time.endswith('ms'):
-                _time = round(float(time.strip('ms')) / 1000, 5)
-            else:
-                _time = round(float(time.strip('ms')), 5)
+            service = None
+            time = 0
+            for x in c.split():
```

Comment:
Could this code be simplified?  Does it make sense to use float/round on numbers that are ints?

Reply:
I simplified it down and removed the unneeded rounds, and switched the floats to ints where needed.